docs/UserGuide.md

```diff
@@ -3,436 +3,713 @@ layout: page
+| `rn/`    | `remark new`     |
+
+> 💡 **Pro Tip:**
+> 
+> Flags are typically derived from the first letter of their corresponding data type (e.g., `n/` for `name`), making them easy to remember!
+
+## 4.4 Arguments
+
+Arguments are the values that follow each flag in a command. **Arguments cannot be empty**, and each must meet specific parsing and format requirements to ensure proper execution of the command.
+
+Refer to the table below for more details.
+
+| **Flag** | **Expected Argument** | **Description**                                | **Requirements**                                                                                | **Case Sensitivity**  |
+|----------|-----------------------|------------------------------------------------|-------------------------------------------------------------------------------------------------|-----------------------|
+| `n/`     | `<NAME>`              | The client's full name                         | Letters, numbers, and spaces only                                                               | ❌                     |
+| `p/`     | `<PHONE>`             | The client's phone number                      | Valid Singapore phone number:<br/> • 8-digit number<br/> • Starts with 8 or 9                   | ❌                     |
+| `e/`     | `<EMAIL>`             | The client's email address                     | Valid email format (`username@domain.com`)                                                      | ❌                     |
+| `a/`     | `<ADDRESS>`           | The client's physical address                  | No specific restrictions                                                                        | ❌                     |
+| `j/`     | `<JOBNAME>`           | The client's job title or profession           | No specific restrictions                                                                        | ❌                     |
+| `i/`     | `<INCOME>`            | The client's annual income                     | Positive number or zero <br/> • Cannot include commas and decimal points<br/> • Must be numeric | ❌                     |
+| `t/`     | `<TIER>`              | The client's assigned tier level               | Predefined tiers:<br/> • Gold, Silver, Bronze, Reject                                           | ✔️                    |
+| `r/`     | `<REMARK>`            | General remarks about the client               | No specific restrictions                                                                        | ❌                     |
+| `ra/`    | `<REMARK TO APPEND>`  | Append information to the existing remark      | No specific restrictions                                                                        | ❌                     |
+| `rn/`    | `<NEW REMARK>`        | Replaces the existing remark with a new remark | No specific restrictions                                                                        | ❌                     |    
```

Comment:
Perhaps in line with what pleng suggested, don't state that there are `no specific restrictions` rather just state `any combination of characters` but state what they can put first before stating what they cannot put.